packages/desktop/Cargo.toml

```diff
@@ -51,6 +51,7 @@ libc = "0.2.174"
 rand = { workspace = true, features = ["std_rng"] }
 subtle = { version = "2.6", features = ["const-generics"] }
 webbrowser = { version = "1.0" }
+url = { workspace = true }
```

Comment:
This is already a dependency of wry itself, so this shouldn't add to the total dependency count